torchmdnet/models/utils.py

```diff
@@ -76,6 +77,143 @@ def forward(self, z, x, edge_index, edge_weight, edge_attr):
     def message(self, x_j, W):
         return x_j * W
 
+import torchmdnet.neighbors as neighbors
+class DistanceCellList(torch.nn.Module):
```

Comment:
It can run 3 strategies and just one of them is the cell list. Maybe `OptimizedDistance`?

Reply:
Its better than the current name, so I am using this :P.